### PR TITLE
UTY-1105: Refined error message

### PR DESCRIPTION
**Pull request by JonasImprobable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Describe your changes here.
#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?
#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs [hunk 1]
```diff
@@ -24,7 +24,8 @@ public class RequiredFieldInjector
 
         private const string LoggerName = nameof(RequiredFieldInjector);
         private const string BadRequiredMemberWarning
-            = "[Require] attribute found on member that is not Injectable. This member will be ignored.";
+            = "[Require] attribute found on member that is not Injectable. This member will be ignored. " 
+            + "Please make sure that types marked with [Require] are located in the <component name>.Requirable or WorldsCommands.Requirable namespace and are not part of the ECS API of the SpatialOS GDK by accident.";
```
Comment (callumlawson) [inline workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs]: i'd suggest removing "by accident"

Comment (JonasImprobable) [reply workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs]: Removed.

File: workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs [hunk 2]
```diff
@@ -24,7 +24,8 @@ public class RequiredFieldInjector
 
         private const string LoggerName = nameof(RequiredFieldInjector);
         private const string BadRequiredMemberWarning
-            = "[Require] attribute found on member that is not Injectable. This member will be ignored.";
+            = "[Require] attribute found on member that is not Injectable. This member will be ignored. " 
+            + "Please make sure that types marked with [Require] are located in the <component name>.Requirable or WorldsCommands.Requirable namespace and are not part of the ECS API of the SpatialOS GDK.";
```
Comment (firtina-improbable) [inline workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs]: I think " and are not part of the ECS API of the SpatialOS GDK." is impossible when combined with the previous part of the sentence

Comment (firtina-improbable) [reply workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs]: I mean, is it possible for a part of `.Requirable` to be non-requirable?

Comment (JonasImprobable) [reply workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs]: Removed the second part.

**Pull request merged**